dev/storage_base.h

```diff
@@ -764,6 +755,30 @@ namespace sqlite_orm {
                 return res;
             }
 
+            std::vector<std::string> object_names(const std::string& type) {
```

Comment:
nit: one can accept `std::string_view` instead of `std::string` to reduce amount of dynamic allocations

Reply:
Let's choose `string_constant_type`: `std::vector<std::string> object_names(string_constant_type type)`